DecipherNow/gm-fabric-dashboard
# Refactor App to use Slugs

Note: Refactor currently breaks tons of tests. The mock JSON data needs to get refactored.

Resolves #805 
Resolves #1437 

## Proposed Changes

  - Generate slugs when we inject services from the discovery-service using the service name and version
  - Key services in Redux off the slug
  - Refactor Routers to collapse service name and version into this new slug
  
## How to Test

  - Run app and check for regressions
  - Run storybook and check for regressions
